### PR TITLE
Update plugins.json

### PR DESCRIPTION
**Pull request by zkwokleung**
Added a plugin called "Wallpaper Searcher"

### CODE REVIEW

**Comment by VictoriousRaptor** [comment]
Hi, please add a github action to your plugin repo to automatically publish releases.

**Comment by zkwokleung** [comment]
Hi, thank you for reminding me. It's done.

File: plugins.json [hunk 1]
```diff
@@ -2317,7 +2317,7 @@
         "UrlSourceCode": "https://github.com/Jwiggiff/Flow-Launcher-Timezone-Conversion-Plugin",
         "IcoPath": "https://cdn.jsdelivr.net/gh/Jwiggiff/Flow-Launcher-Timezone-Conversion-Plugin@main/assets/icon.png",
         "DateAdded": "2024-05-17T06:12:12Z",
-        "LatestReleaseDate": "2024-05-24T03:13:31Z"
+        "LatestReleaseDate": "2024-05-16T02:53:07Z"
```
Comment (jjw24) [inline plugins.json:2320]:
```suggestion
        "LatestReleaseDate": "2024-05-24T03:13:31Z"
```

**Review by jjw24 — DISMISSED** [review]

File: plugins.json [hunk 2]
```diff
@@ -2334,4 +2334,16 @@
...
+        "Name": "Wallpaper Searcher",
+        "Description": "A plugin for quickly searching for wallpapers",
+        "Author": "zkwokleung",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/zkwokleung/Flow.Launcher.Plugin.WallpaperSearcher",
+        "UrlDownload": "https://github.com/zkwokleung/Flow.Launcher.Plugin.WallpaperSearcher/releases/download/v1.0.0/Flow.Launcher.Plugin.WallpaperSearcher.zip",
+        "UrlSourceCode": "https://github.com/zkwokleung/Flow.Launcher.Plugin.WallpaperSearcher",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/zkwokleung/Flow.Launcher.Plugin.WallpaperSearcher@main/Images/icon.png"
+    },
```
Comment (jjw24) [inline plugins.json:2348]:
```suggestion
    }
```

File: plugins.json [hunk 3]
```diff
@@ -2334,4 +2334,16 @@
         "DateAdded": "2024-05-29T12:24:56Z",
         "LatestReleaseDate": "2024-05-22T15:41:39Z"
     }
-]
\ No newline at end of file
+    {
```
Comment (jjw24) [inline plugins.json:2337]:
```suggestion
    ,{
```

**Review by jjw24 — APPROVED** [review]

**Pull request merged**